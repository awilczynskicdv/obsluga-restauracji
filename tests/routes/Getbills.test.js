import { callApi } from "../helpers.js";

const PATH = `api/private/bills`;

test(`[GET] ${PATH} - valid request - should return status 200`, async () => {
  const response = await callApi("get", "/bills/all", {}, {}, false);
 expect(response.data).toMatchSnapshot();
 expect(response.status).toEqual(200);
});
