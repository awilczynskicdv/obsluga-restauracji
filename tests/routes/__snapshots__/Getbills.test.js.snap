// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[GET] api/private/bills - valid request - should return status 200 1`] = `
Object {
  "data": Array [
    Object {
      "currency": "EUR",
      "orderId": 1,
      "value_currency": 33.64,
      "value_pln": 33.64,
    },
    Object {
      "currency": "USD",
      "orderId": 2,
      "value_currency": 7.64,
      "value_pln": 20.64,
    },
  ],
}
`;
